sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -102,8 +109,7 @@ supports conversion of values of `float` type to `bfloat16` and back.
 If the device doesn't have the aspect, objects of `bfloat16` class must not be
 used in the device code.
 
-**NOTE**: The `ext_oneapi_bfloat16` aspect is not yet supported.  The
-`bfloat16` class is currently supported only on Xe HP GPU and Nvidia GPUs with Compute Capability >= SM80.
+**NOTE**: The `bfloat16` class is currently supported only on Xe HP GPUs and Nvidia GPUs with Compute Capability >= SM80.
 
 == New `bfloat16` class
```

Comment:
Oh, I just noticed this.  This section and the other sections that form the specification should be sub-sections of "Specification" (i.e. a peer of "Feature test macro").  The "Issues" and "Revision History" sections below should **not** be sub-sections under "Specification".